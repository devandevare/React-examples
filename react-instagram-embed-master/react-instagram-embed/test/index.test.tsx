import * as React from 'react';
import renderer from 'react-test-renderer';
import InstagramEmbed from '../src';

test('should render self and sub-components', () => {
  const tree = renderer
    .create(<InstagramEmbed url="https://instagr.am/p/Zw9o4/" clientAccessToken="abc|123" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
