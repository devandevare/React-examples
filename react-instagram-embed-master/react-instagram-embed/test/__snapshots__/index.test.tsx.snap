// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render self and sub-components 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "",
    }
  }
/>
`;
